### ISSUE TITLE
VideoTrack.detach() freezes on the last frame

### ISSUE THREAD

**Issue by KaelWD** [twilio/twilio-video.js#791]
- [x] I have verified that the issue occurs with the latest twilio-video.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-video.js/blob/master/CHANGELOG.md).
 - [x] I reviewed the [Common Issues](https://github.com/twilio/twilio-video.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [ ] I verified that the [Quickstart](https://github.com/twilio/video-quickstart-js) application works in my environment.
 - [x] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

**Code to reproduce the issue:**

```js
room.on('trackUnsubscribed', (track) => {
  if (track.kind === 'video') {
    track.detach()
  }
})
```

**Expected behavior:**

The video element should go blank

**Actual behavior:**

The video freezes on the last displayed frame

Calling `videoElement.srcObject = null` instead has the expected result
EDIT: or `track.detach().forEach(el => { el.srcObject = null })`

**Software versions:**

 - [ ] Browser(s): Chrome 78.0.3904.70
 - [ ] Operating System: Kubuntu 18.04
 - [ ] twilio-video.js: 2.0.0-beta15
 - [ ] Third-party libraries (e.g., Angular, React, etc.): Vue 2.6, Vuetify 1.5, Nuxt 2.10

**Comment by makarandp0** [comment]
Hey @KaelWD, Thank you for writing about this issue - looks like you have already found a workaround, and are unblocked right?
 
But I will file go ahead and file a bug in our database to handle this from SDK side.
Thanks,
Makarand
JSDK Team

**Comment by makarandp0** [comment]
Hey @KaelWD, I am closing this old issue. We are keeping the ticket tracking in our internal database open, but it is not very high priority. (VIDEO-3589).
